python/ray/train/torch.py

```diff
@@ -177,6 +177,9 @@ def __init__(self, base_dataloader: DataLoader, device: torch.device):
         self._dataloader = base_dataloader
         self._device = device
 
+    def _move_to_device(self, item):
+        return tuple(i.to(self.device) for i in item)
```

Comment:
I'm not sure if this will ever happen, but what if an item in the tuple is not a tensor?

Reply:
Also, can the device ever be None?

Reply:
See comments above- these are addressed.